microsoft-edge/devtools-guide-chromium/whats-new/2021/05/devtools.md

```diff
@@ -128,29 +162,192 @@ Version 1.1.8 of the [Microsoft Edge Developer Tools for Visual Studio Code][Vis
+
+For example, to produce the results shown in the previous screenshot:
+
+1.  In DevTools version 92 or later, in the **Console**, enter the following statement and then select `Enter`.
+
+    ```javascript
+    const a = 'declare';
+    ```
+    
+1.  Enter the following statement and then select `Enter`.  This is a separate REPL script, independent from the above script, so no error message appears.
+
+    ```javascript
+    const a = 'redeclare';
+    ```
+    
+1.  Enter both of the following statements as a single REPL script.  To do this, either copy and paste the following code listing into the **Console**, or type the first statement followed by `Shift`+`Enter` and then type the second statement followed by `Enter`.  The const variable is redeclared within the same REPL script, so now an error message appears.
+
+    ```javascript
+    const b = 'declare';
+    const b = 'no redeclaration on same REPL script';
+    ```
+    
+To review the history of this feature in the Chromium open-source project, navigate to Issue [1076427][CR1076427].
+
+
+### New shortcut to view iframe details
+
+To quickly view `iframe` details, you can now right-click an `iframe` element in the **Elements** tool, and then select **Show iframe details**.
+
+:::image type="complex" source="../../media/2021/05/show-iframe-details.msft.png" alt-text="iframe details view" lightbox="../../media/2021/05/show-iframe-details.msft.png":::
+   iframe details view
+:::image-end:::
+
+This displays the details about the `iframe` in the **Application** tool.  In the **Application** tool, you can examine document details, security and isolation status, permissions policy, and more, to debug potential issues.
+
+:::image type="complex" source="../../media/2021/05/show-iframe-details-application-tool.msft.png" alt-text="Frame details in the Application tool" lightbox="../../media/2021/05/show-iframe-details-application-tool.msft.png":::
+   Frame details in the **Application** tool
+:::image-end:::
+
+<!-- demo page: https://wolfib.github.io/web-demos/ esp https://wolfib.github.io/web-demos/jsIframe.html -->
+
+To review the history of this feature in the Chromium open-source project, navigate to Issue [1192084][CR1192084].
+
+
+### Enhanced CORS debugging support
+
+Cross-origin resource sharing (CORS) errors are now surfaced in the **Issues** tab.  There are various potential causes of CORS errors.  Click to expand each issue to understand the potential causes and solutions.
+
+:::image type="complex" source="../../media/2021/05/cors-debugging-support.msft.png" alt-text="CORS issues in the Issues tab" lightbox="../../media/2021/05/cors-debugging-support.msft.png":::
+   CORS issues in the Issues tab
+:::image-end:::
+
+<!-- screenshot uses http://cors-errors.glitch.me -->
+
+To review the history of this feature in the Chromium open-source project, navigate to Issue [1141824][CR1141824].
+
+
+### Renamed XHR label to Fetch\/XHR
```

Comment:
```suggestion
### Renamed XHR filter to Fetch\/XHR
```

Maybe for consistency of what these "labels" do, we should just call them "filter" instead. Selecting them filters the Network Activity Log down to only those that match the selected filter.